src/views/Home.jsx

```diff
@@ -7,10 +8,14 @@ export function Home({ data, setListPath }) {
 				Hello from the home (<code>/</code>) page!
 			</p>
 			<ul>
-				{/**
-				 * TODO: write some JavaScript that renders the `lists` array
-				 * so we can see which lists the user has access to.
-				 */}
+				{data.map((list, i) => (
+					<SingleList
+						key={i}
+						name={list.name}
+						path={list.path}
+						setListPath={setListPath}
+					/>
+				))}
```

Comment:
Nice work here team! This is an elegant code and kudos for using the `key` too 💥